compass_sdk/root_rbac.py

```diff
@@ -0,0 +1,131 @@
+import json
+from typing import List
+
+import requests
+
+from compass_sdk.types import (
+    GroupCreateRequest,
+    GroupCreateResponse,
+    GroupFetchResponse,
+    GroupUserDeleteResponse,
+    PolicyRequest,
+    RoleCreateRequest,
+    RoleCreateResponse,
+    RoleDeleteResponse,
+    RoleFetchResponse,
+    RoleMappingDeleteResponse,
+    RoleMappingRequest,
+    RoleMappingResponse,
+    UserCreateRequest,
+    UserCreateResponse,
+    UserDeleteResponse,
+    UserFetchResponse,
+)
+
+
+def create_root_user(compass_url: str, tenancy_token: str) -> UserCreateResponse:
+    headers = {"Authorization": f"Bearer {tenancy_token}", "Content-Type": "application/json"}
+    response = requests.post(f"{compass_url}/security/admin/rbac/v1/root", headers=headers)
```

Comment:
I think we probably want to include /api/ before security here, or user has to pass it in themselves.

Reply:
So I do think nginx is expected in dev now that we're back to `run_docker.sh` - but I also agree this is a one-off, better to be handled with a script than the SDK